Fixed #31685 -- Added support for updating conflicts to QuerySet.bulk_create(). (#13065)
django/django: docs/ref/models/querysets.txt

``bulk_create()``
~~~~~~~~~~~~~~~~~

> .. method:: bulk_create(objs, batch_size=None, ignore_conflicts=False, update_conflicts=False, unique_fields=[], update_fields=[])

Reviewer: This should match the signature:
```suggestion
.. method:: bulk_create(objs, batch_size=None, ignore_conflicts=False, update_conflicts=False, update_fields=None, unique_fields=None)
```

where the default is such that at most 999 variables per query are used.

On databases that support it (all but Oracle), setting the ``ignore_conflicts``

> or ``update_conflicts`` parameter to ``True`` tells the database to ignore failure
> or update to insert any rows that fail constraints such as duplicate unique values. 
> Enabling this parameter disables setting the primary key on each model instance 
> (if the database normally supports it).

Reviewer 2: Great work! Just some documentation suggestions:

```suggestion
Setting the ``ignore_conflicts`` parameter to ``True`` tells the database to
ignore failure to insert any rows that fail constraints such as duplicate unique
values.

Setting the ``update_conflicts`` parameter to ``True`` will apply an upsert
where a conflicting object's non-unique fields is used to update the row
which triggered the conflict.

Enabling the ``ignore_conflicts`` or ``update_conflicts`` parameters disable
setting the primary key on each model instance (if the database normally
support it).

.. admonition:: Oracle

    ``ignore_conflicts`` and ``update_conflicts`` is not supported on Oracle.
```

Author: Thanks for your suggestion.

where the default is such that at most 999 variables per query are used.

On databases that support it (all but Oracle), setting the ``ignore_conflicts``
Setting the ``ignore_conflicts`` parameter to ``True`` tells the database to
ignore failure to insert any rows that fail constraints such as duplicate unique
values.

Setting the ``update_conflicts`` parameter to ``True`` will apply an upsert
where a conflicting object's non-unique fields is used to update the row
which triggered the conflict.

Enabling the ``ignore_conflicts`` or ``update_conflicts`` parameters disable
setting the primary key on each model instance (if the database normally
support it).

> .. admonition:: Oracle

Reviewer 2: `versionchanged` should be added. Something like:

```
    .. versionchanged:: 3.2

        The ``update_conflicts `` parameter was added.
```

The 3.2 release notes should also be updated.

where the default is such that at most 999 variables per query are used.

On databases that support it (all but Oracle), setting the ``ignore_conflicts``
Setting the ``ignore_conflicts`` parameter to ``True`` tells the database to
ignore failure to insert any rows that fail constraints such as duplicate unique
values.

Setting the ``update_conflicts`` parameter to ``True``,
``unique_fields`` parameter to which fields are unique and
``update_fields`` parameter to which fields you want to update,
while on conflict it will update ``update_fields`` by ``unique_fields``.

> * If you use mysql as your database,

Reviewer: Please use official names in docs:
- `mysql` -> `MySQL`,
- `postgresql` -> `PostgreSQL`.

  you need to set update_conflicts to True and which fields you want to update.
  And it will update automatically.

> * If you use postgresql and SQLite(>= 3.24.0) as your database,

Reviewer: `SQLite(>= 3.24.0)` -> `SQLite 3.24+`

>   you need to set update_conflicts to True, specify which fields are unique(by constraints) and

Reviewer: backticks are missing 
```suggestion
  you need to set ``update_conflicts`` to ``True``, specify which fields are unique (by constraints) and
```
